inflection/tools/dictionary-parser/README.md

```diff
@@ -1,27 +1,24 @@
 <!--
+ Copyright 2025 Unicode Incorporated and others. All rights reserved.
  Copyright 2021-2024 Apple Inc. All rights reserved.
 -->
 # Dictionary Parsers
 
 These tools generate files that describes the grammatical properties of words from the various sources.
 
-## Usage for Wiktionary
+## Usage for Wikidata
 
-1) Download a copy of Wiktionary data from http://dumps.wikimedia.org/ (e.g. http://dumps.wikimedia.org/enwiktionary/20130825/enwiktionary-20130825-pages-articles.xml.bz2)
+1) Download a copy of Wiktionary data from https://dumps.wikimedia.org/wikidatawiki/entities/ (e.g. https://dumps.wikimedia.org/wikidatawiki/entities/20250115/wikidata-20250115-lexemes.json.bz2)
 2) Decompress dump
-3) Run `./ParseWiktionary ~/Downloads/enwiktionary-20130825-pages-articles.xml > ../../resources/com/apple/inflection/dictionary/dictionary_en.lst`
-
-## Usage for Apple's Lexical Resources (ALR)
-
-1) Get the latest lexicon for your language.
```

Comment:
Is this from step 3 in previous section?

Reply:
This tool was used to consume 2 formats. Now it's only 1 (Wikidata), but I see the typos in these steps.  I'll fix that.

Reply:
The instructions have been fixed. Please check them again.